### PR TITLE
Add code example for environment module

### PR DESCRIPTION
**Pull request by Wyfy0107**

### CODE REVIEW

File: content/code-examples/environments/create-environment.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Novu } from '@novu/node';
+
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+
+const payload: IEnvironmentCreatePayload = {
+  name: 'test',
+  parentId: 'as20dfbjsfi3ssdk39dh'
+} 
+
+await novu.environments.create(payload);
```
Comment (florian-lefebvre) [inline content/code-examples/environments/create-environment.md:17]:
```suggestion
await novu.environments.create({
  name: 'test',
  parentId: 'as20dfbjsfi3ssdk39dh'
});
```

File: content/code-examples/environments/update-environment.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const payload: IEnvironmentUpdatePayload = {
+  name: 'new environment';
+  identifier: 'test env';
+  parentId: '109ufdsnfkhk890hhjsdf';
+}
+
+const envId = '12390dsfkhjk9dfd'
+
+await novu.environments.updateOne(envId, payload);
```
Comment (florian-lefebvre) [inline content/code-examples/environments/update-environment.md:20]:
```suggestion
await novu.environments.updateOne(envId, {
  name: 'new environment';
  identifier: 'test env';
  parentId: '109ufdsnfkhk890hhjsdf';
});
```

File: content/code-examples/environments/update-widget.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+```javascript label=Node.js
+import { Novu } from '@novu/node';
+
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+
+const payload: IWidgetUpdatePayload = {
+  notificationCenterEncryption: true
+}
+
+await novu.environments.updateWidget(payload);
```
Comment (florian-lefebvre) [inline content/code-examples/environments/update-widget.md:16]:
```suggestion
await novu.environments.updateWidget({
  notificationCenterEncryption: true
});
```

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
Looks great! I requested changes to keep docs consistent. Ping scopsy once you've made the changes 👍

File: content/code-examples/environments/create-environment.md (hunk 1 above)
Comment (p-fernandez) [reply content/code-examples/environments/create-environment.md:17]: I do not find that bad to separate the payload. It also adds the type of the payload that helps as a referral to check in the code the potential object.

File: content/code-examples/environments/update-environment.md (hunk 2 above)
Comment (p-fernandez) [reply content/code-examples/environments/update-environment.md:20]: Same reasoning as here: https://github.com/novuhq/api-docs/pull/22/files#r1020280331

File: content/code-examples/environments/update-widget.md (hunk 3 above)
Comment (p-fernandez) [reply content/code-examples/environments/update-widget.md:16]: And again. Maybe a different point of view I have. 🙂

**Review by p-fernandez — APPROVED** [review]

File: content/code-examples/environments/create-environment.md (hunk 1 above)
Comment (florian-lefebvre) [reply content/code-examples/environments/create-environment.md:17]: No problem, but we should do it in the rest of the docs then

**Comment by scopsy** [comment]
I tend to aggree with @p-fernandez that the payload in a separated way is great because we could also optionally annotate the actual payload interface for easier usage in the future. So think we can merge this one

**Pull request merged**